src/lib/services/api-token-service.ts

```diff
@@ -102,12 +102,16 @@ export class ApiTokenService {
     }
 
     public getUserForToken(secret: string): ApiUser | undefined {
-        let token = this.activeTokens.find((t) => t.secret === secret);
+        let token = this.activeTokens.find(
+            (activeToken) => activeToken.secret === secret,
+        );
 
         // If the token is not found, try to find it in the legacy format with the metadata alias
         // This is to ensure that previous proxies we set up for our customers continue working
```

Comment:
Could remove `metadata` from this comment, as well as "customers" since this could be a generic feature.

Reply:
Thanks